tests/modules/programs/alacritty/example-settings-expected.toml

```diff
@@ -0,0 +1,8 @@
+[[keyboard.bindings]]
+chars = "\x0c"
```

Comment:
TOML [doesn't support](https://toml.io/en/v1.0.0#string) this escape sequence, and instead wants us to use `\u000c`, which YAML doesn't support.

It's really a shame that the user should have to care about such format-specific bureaucracy, given the whole point of managing your configuration with Nix...

But anyway, this should at least be mentioned in the example for lack of a better solution.

Reply:
...or maybe it does, since it's a superset of JSON.

Anyway, the following seems to work independently of the format and doesn't require any backslash substitution:

```nix
keyboard.bindings = let
 c = (builtins.fromTOML ''c = "\u000c"'').c;
in [ {
  key = "K";
  mods = "Control";
  chars = c;
} ];
```

This could probably be turned into a library function `lib.hexByte` as @infinisil suggested [here](https://github.com/NixOS/nixpkgs/issues/97310#issuecomment-687902805).

Reply:
Good point! https://github.com/nix-community/home-manager/pull/4853

Reply:
Oh that might be a better approach, but a bit heavy without the library function. I don't like the substitution in the buildCommand at all though.